.github/workflows/benchmarks.yml

```diff
@@ -62,6 +62,12 @@ jobs:
           cargo bench --bench merkle_tree -- --output-format bencher | tee -a ../../output.txt
           cd ../..
           
+      - name: Benchmark console/types
+        run: |
+          cd console/types
+          cargo bench --bench merkle_tree -- --output-format bencher | tee -a ../../output.txt
```

Comment:
seems like this one needs to be adjusted

Reply:
good catch